src/libponyrt/actor/actor.c

```diff
@@ -230,10 +233,55 @@ static void try_gc(pony_ctx_t* ctx, pony_actor_t* actor)
   DTRACE1(GC_END, (uintptr_t)ctx->scheduler);
 }
 
-bool ponyint_actor_run(pony_ctx_t* ctx, pony_actor_t* actor, size_t batch)
+// return true if mute occurs
```

Comment:
its really hard to see what has changed here because of the reordering of methods. im going to need to check out side by side and compare old and new.

Reply:
Understood. The intent was to reorganize some of the logic into re-usable functions so hopefully there aren't any actual logic changes included.